frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -87,4 +87,19 @@ export async function writeActivity(tripId, activityId, newValues) {
     console.log(e);
     return false;
   }
+}
+
+/**
+ * Get the value of a reference. 
+ * 
+ * @param {Reference} ref Reference to get the value of.
+ * @param {string} ignoreValue The "null" or "none" value that ref could be.
+ * @param {string} defaultValue Value to return if ref.current.value === ignoreValue.
+ * @returns defaultValue if ref.current.value === ignoreValue, else ref.current.value.
+ */
+export function getRefValue(ref, ignoreValue, defaultValue=null) {
+  if (ref.current.value === ignoreValue) {
+    return defaultValue;
+  } 
+  return ref.current.value;
```

Comment:
I really like the use of this function and `getField` to do the input cleaning/formatting!